Tests/Unit/NetworkServer/PrometheusMetricExporterTests.cs

```diff
@@ -117,6 +125,25 @@ public void When_Histogram_Series_Is_Recorded_Should_Export_To_Prometheus()
                 this.recordHistogramMock.Verify(c => c.Invoke(Histogram.Name, new[] { gatewayId }, value), Times.Once);
         }
 
+        [Fact]
+        public async void When_ObservableGauge_Is_Recorded_Should_Export_To_Prometheus()
+        {
+            // arrange
+            var observeValue = new Mock<Func<Measurement<int>>>();
+            var stationEui = new StationEui(1);
+            var measurement = new Measurement<int>(1, KeyValuePair.Create(MetricRegistry.GatewayIdTagName, (object?)stationEui));
+            observeValue.Setup(ov => ov.Invoke()).Returns(measurement);
+            using var meter = new Meter("LoRaWan", "1.0");
+            var observableGauge = meter.CreateObservableGauge(ObservableGauge.Name, observeValue.Object);
```

Comment:
Check codeql warning

Reply:
Thanks for the hint, since the CodeQL warnings break the build I usually fix those before publishing the PR.